backend/src/nodes/properties/outputs/pytorch_outputs.py

```diff
@@ -1,10 +1,51 @@
-from .. import expression
+try:
+    import torch
+except:
+    torch = None
+
+from typing import Union
+
+from ...properties import expression
 from .base_output import BaseOutput
 
+try:
+    from ...utils.architecture.RRDB import RRDBNet as ESRGAN
+    from ...utils.architecture.SPSR import SPSRNet as SPSR
+    from ...utils.architecture.SRVGG import SRVGGNetCompact as RealESRGANv2
+    from ...utils.architecture.SwiftSRGAN import Generator as SwiftSRGAN
+
+    PyTorchModel = Union[RealESRGANv2, SPSR, SwiftSRGAN, ESRGAN]
+except:
+    PyTorchModel = None
+
+
+class ModelOutput(BaseOutput):
+    def __init__(
+        self,
+        model_type: expression.ExpressionJson = "PyTorchModel",
+        label: str = "Model",
+    ):
+        super().__init__(model_type, label)
+
+    def broadcast(self, value: PyTorchModel) -> dict:  # type: ignore
```

Comment:
Not typing the return type would probably be better here, since pyright and co could at least infer the type then.